docs/guides/transfer-warp-route-ownership.mdx

```diff
@@ -7,40 +7,35 @@ One of the quickest way to transfer a warp route ownership is by using the [Hype
 ## Prerequisites
 
 - The warp route config generated by `hyperlane warp init`.
-    - This config is used to deploy a warp route. By default, it takes the filepath of `CURRENT_DIR/warp-route-deployment.yaml`.
-- The warp route deployment artifacts generated by `hyperlane warp deploy`.
-    - This deployment artifact file is generated by a warp route deployment. By default, it is written to `$HOME/.hyperlane/deployments/warp_routes/TOKEN/chain1-chain2-config.yaml`.
+    - This config is used to deploy a warp route. By default, it takes the filepath of `CURRENT_DIR/configs/warp-route-deployment.yaml`.
+- The warp route token symbol used in `hyperlane warp deploy`.
+    - The token symbol can be found onchain, or in your local Hyperlane Registry. By default, it is written to `$HOME/.hyperlane/deployments/warp_routes/<tokenSymbol>`.
 - Access to the private key that currently owns the warp route.
 
 :::info
 If you followed the [Deploy a Warp Route](https://docs.hyperlane.xyz/docs/guides/deploy-warp-route) guide, you may have deployed a warp route with the owner set to the single private key. In production, it is advisable to use a multisig.
 :::
 
-To confirm using the Hyperlane CLI, locate your warp route under the `/deployments/warp_routes` directory, and execute the following with `address` set to the `addressOrDenom` of your warp route:
+To confirm using the Hyperlane CLI, run the folllowing command with your token symbol and the chain it is deployed on:
 
 ```bash
-hyperlane warp read --address <addressOrDenom> --chain yourchain
+hyperlane warp read --symbol <tokenSymbol> --chain <yourchain>
 ```
 
+You should be able to select the warp route that you wish to update the ownership for.
```

Comment:
If all this is duplicated, should it be in a shared place, like `docs/partials` or `src/components`?

Reply:
Moved most duplicates to partials